docs/user/alerting/action-types/pagerduty.asciidoc

```diff
@@ -150,32 +150,19 @@ Integration Key::   A 32 character PagerDuty Integration Key for an integration
      routingKey: testroutingkey
 --
 
-[[pagerduty-connector-config-properties]]
-**`config`** defines the action type specific to the configuration and contains the following properties:
+Config defines information for the action type.
```

Comment:
Should the order of the Pager Duty sections be to match the template?

- Connector configuration
- Action configuration
- Configure PagerDuty 

Also, should the three bullet points at the top of the doc be removed?

Reply:
There was a lot of additional information in the PagerDuty doc that I didn't want to just remove. I will try to find out whether it should stay or not.

Reply:
@gchaps Updated to follow your suggestions.